Assignments/Program.cs

```diff
@@ -1,3 +1,75 @@
-﻿using System;
+﻿// ---------------------------------------------------------------------------------------
+// Spark23 Assignments
+// Copyright (c) Metamation India.
+// ---------------------------------------------------------------------------------------
+// Program.cs
+// T18 - The Armstrong Number
+// ---------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using static System.ConsoleColor;
+
+namespace Spark;
+public class T18 {
+   public static void Main () {
+      List<int> task1 = new () { 0, 1, 2, 5, 9, 10, 15, 20, 25, 27 },

```

Comment:
```suggestion
      List<int> task = new () { 0, 1, 2, 5, 9, 10, 15, 20, 25, 27 },

```
Usually we don't use 1. It should task and task2